debian/systemd.install

```diff
@@ -17,6 +17,7 @@ lib/udev/rules.d/71-seat.rules
 lib/udev/rules.d/99-systemd.rules
 usr/bin/systemd-cgls
 usr/bin/systemd-cgtop
+usr/bin/systemd-coredumpctl
```

Comment:
I'm pretty sure this will be `lib/systemd/systemd-coredumpctl` since it's specified in the Makefile as `rootlibexec_PROGRAMS`. In that case, it'll be included by the catch all `lib/systemd/` above.